engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/fenced-pages.factory.js

```diff
@@ -0,0 +1,28 @@
+angular.module('Bastion.organizations').service('FencedPages',
+    [function () {
+        var fencedPages = [
+            'products',
+            'activation-keys',
+            'environments',
+            'subscriptions',
+            'gpg-keys',
+            'sync-plans',
+            'content-views',
+            'errata',
+            'content-hosts',
+            'host-collections'
+        ];
+
+        return {
+            addPages: function () {
+                var i;
+                for (i = 0; i < arguments.length; i++) {
```

Comment:
Why use arguments here instead of a parameter passed into the function?  You can also use `angular.forEach` to iterate over the array.